app/code/community/Vindi/Subscription/Block/Form/Cc.php

```diff
@@ -153,7 +153,7 @@ protected function isSingleQuote($quote)
     /**
      * @return Vindi_Subscription_Helper_API
      */
-    private function api()
+    protected function api()
```

Comment:
Da mesma forma que a chave, vale expormos `api()`? Aqui a chave em si não sai, mas com este método se faz muito.

Reply:
Em um cenário hipotético se for realizado override de qualquer outro method protected ou public da class, gerará erro na call do $this->api() como private, que faz parte apenas do parent